docs/projects/index.md

```diff
@@ -37,6 +37,11 @@ but are sometimes forgotten. Read more about [the project here]({{ site.url }}{{
 The Transparent Funding project is an effort to encourage projects and communities to share how to financially contribute to their projects, and given some financial contribution to provide disclosure about how the support is invested. Read more about [the project here]({{ site.url }}{{ site.baseurl }}/docs/projects/funding/).
 
 
+## Searchable Linkable Open Public Indexed (SLOPI) Communication
+
+The Searchable Linkable Open Public Indexed (SLOPI) Communication project is an effort to remind open source developers of the importance of transparent communication rather than favoring Slack for discussion and decision making. Read more about [the project here]({{ site.url }}{{ site.baseurl }}/docs/projects/slopi-communication/).
+
```

Comment:
With this point you make the goal seem like it's explicitly targeted at Slack, which I don't think you want to do. SLOPI is an effort to remind open source developers the importance of transparent communication, and Slack is an example of (potentially) not open communication if funds aren't invested to make archives available, and the boards aren't entirely open.

Reply:
Well, Slack is the elephant in the room so I thought I'd call it out specifically, but I take your point. What if I change it to "tools like Slack"?

Reply:
yes that would work! Can you think of one other example so it's not singled out?